plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/FilterMediatorPropertiesEditionPartForm.java

```diff
@@ -783,6 +782,18 @@ public void focusGained(FocusEvent e) {
         sourceElements = new Control[] { itemLabel, sourceText, itemHelp };
         return parent;
     }
+    
+    private void openNamespacedPropertyEditor(final Composite parent) {
+        EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(),
+                SWT.NULL, source);
+        // valueExpression.setPropertyValue(valueExpressionText.getText());
```

Comment:
Remove commented code.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/commit/1d53e06b88e566a6d316666c06dd48202d396d30